src/services/workBench.js

```diff
@@ -0,0 +1,22 @@
+import { kfVariantClusterUrl } from 'common/injectGlobals';
+import { initializeApi } from 'services/api';
+
+const api = initializeApi();
+
+export const startCluster = async () =>
+  await api({
+    method: 'POST',
+    url: kfVariantClusterUrl,
+  });
+
+export const stopCluster = async () =>
+  await api({
+    method: 'DELETE',
+    url: kfVariantClusterUrl,
+  });
+
+export const getClusterStatus = async () =>
+  await api({
```

Comment:
Why is it required to block the thread here if it's pass to a thunk?

Reply:
fixed!